### PR TITLE
[DBR-112] Add Dockerfiles for int and qa

### PR DESCRIPTION
**Pull request by fgchaio**
hi @FromDoppler/doppler-webapp team, im adding dockerfiles for int and qa.

### CODE REVIEW

File: Dockerfile.QA [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Build project
+RUN yarn build:qa
+
+# Host project in nginx
+FROM nginx
+WORKDIR /app
+COPY --from=build /app/build /usr/share/nginx/html
+
+ENTRYPOINT ["nginx", "-g", "daemon off;"]
+EXPOSE 80 443
```
Comment (andresmoschini) [inline Dockerfile.QA]: Do you configure the real ports in swarm configuration?

File: Dockerfile.QA [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM node:10 As build
```
Comment (andresmoschini) [inline Dockerfile.QA]: What do you think about adding a comment in each dockerfile file about that all files have duplicated code and should be edited together?

**Comment by andresmoschini** [comment]
❤️

File: Dockerfile.QA (hunk 1 above)
Comment (fgchaio) [reply Dockerfile.QA]: Exactly, we where thinking of adding it to readme or some other file to give some visibility to it.

File: Dockerfile.QA (hunk 2 above)
Comment (fgchaio) [reply Dockerfile.QA]: you are right. Forgot about it thx.

File: Dockerfile [hunk 3]
```diff
@@ -1,3 +1,4 @@
+#Any change to this file should be replecated in dockerfile.int and dockerfile.qa
```
Comment (andresmoschini) [inline Dockerfile]: Very, very minor ;)

```suggestion
# Any change to this file should be replecated in dockerfile.int and dockerfile.qa
```

Comment (fgchaio) [reply Dockerfile]: 😆

**Pull request merged**